### PR TITLE
Fix/timeout warning browser compatibility

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello card](https://trello.com/c/MA4PWU4W/206-show-user-warning-of-session-timeout)

## Context

When you use `require('dialog-polyfill')` the `window.dialogPolyfill` is only available on Chrome.
In order to have the dialogPolyfill object available in all browsers we should be using the "default" from the `require` clause (see code on timeout-warning.js).

Also, the dialog should be under the body and should not be located in nested divs
(see https://github.com/GoogleChrome/dialog-polyfill#limitations for more details)

### CODE REVIEW

File: src/js/component/timeout-warning.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // on https://github.com/alphagov/govuk-design-system-backlog/issues/104
 // and https://github.com/alphagov/govuk-design-system-backlog/issues/103
 // This also were based on the recommended accessible timeout warning.
-require('dialog-polyfill')
+const dialogPolyfill = require('dialog-polyfill')
```
Comment (sequencemedialimited) [inline src/js/component/timeout-warning.js]: Or
```
const { default: dialogPolyfill } = require('dialog-polyfill')
```
Then, below ...

File: src/js/component/timeout-warning.js [hunk 2]
```diff
@@ -135,7 +135,7 @@ TimeoutWarning.prototype.dialogSupported = function () {
   } else {
     // Native dialog is not supported by browser so use polyfill
     try {
-      window.dialogPolyfill.registerDialog(this.$module)
+      dialogPolyfill.default.registerDialog(this.$module)
```
Comment (sequencemedialimited) [inline src/js/component/timeout-warning.js]:
```
dialogPolyfill.registerDialog(this.$module)
```

Comment (tomas-stefano) [reply src/js/component/timeout-warning.js]: Your suggestion sounds much better. Updated. ♻️

File: src/js/component/timeout-warning.js (hunk 1 above)
Comment (tomas-stefano) [reply src/js/component/timeout-warning.js]: Your suggestion sounds much better. Updated. ♻️

**Pull request merged**